test/components/generators/test_hugging_face_api.py

```diff
@@ -315,23 +320,7 @@ def test_run_serverless(self):
         assert len(response["meta"]) > 0
         assert [isinstance(meta, dict) for meta in response["meta"]]
 
-    @pytest.mark.flaky(reruns=5, reruns_delay=5)
-    @pytest.mark.integration
-    @pytest.mark.skip(reason="Temporarily skipped due to weird responses from the selected model.")
-    def test_live_run_streaming_check_completion_start_time(self):
```

Comment:
I combined this with the above integration test since it seemed unnecessary to ping the api twice with the same prompt

Reply:
Never mind I see that this one uses streaming so it is a different test, I'll add it back.